user/deployment/s3.md

```diff
@@ -268,7 +268,20 @@ deploy:
 ```
 {: data-file=".travis.yml"}
 
-### Using S3-compatible Object Storage
+## Set dot_match flag to upload files starting with a period
+
+S3 uploads can be set to upload all files starting with a `.`
```

Comment:
I think "all files starting with a `.`" is somewhat misleading. Namely, it reads as though those are the only files that will be uploaded.

May I suggest changing this to: "all files including those starting with a `.`"?

Reply:
Agreed, let's do that.